src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, serif;
+  color: #3d4e61;
+}
+
+.search-bar {
+  background: #fff;
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-input {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  flex: 1;
+  outline: none;
+}
+
+.search-input:hover {
+  background: #fff;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-Heavy, serif;
+  font-size: 16px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.big-input {
+  line-height: 495%;
```

Comment:
why do you use `line-height` with `%`? use `height` instead

Reply:
Pay attention to this comment, please